packages/components/ng-ovh-payment-method/src/components/choice/template.html

```diff
@@ -11,7 +11,7 @@
     <oui-radio
         data-name="paymentMethod"
         data-model="$ctrl.model"
-        data-value="true"
+        data-value="$ctrl.defaultPaymentMethod"
```

Comment:
This will break the API call done here: https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/private-database/order/clouddb/private-database-order-clouddb.controller.js#L145
The API call at https://github.com/ovh/manager/blob/master/packages/manager/modules/web-universe-components/src/order-cart/order-cart.service.js#L117 expects a boolean for `autoPayWithPreferredPaymentMethod`. Not sure if passing an object would work here.
Similarly, we would have to check out the other places where the `ovh-payment-method-choice` component is used :)

Reply:
Yes you're right @ganeshkumar1989, that's why I've set the commit as BREAKING CHANGE.

Once this PR will be merged, the `@ovh-ux/ng-ovh-payment-method` package version will be increased to `6.0.0`, so it won't break anything until we explicitly use the new version, and make the adaptation ;)

Reply:
Do we have a migration plan for this ?

Reply:
@marie-j you mean adaptation of application using this module ?

Reply:
@marie-j here it is : https://github.com/ovh/manager/pull/2243